### DISCUSSION TITLE
Training SVM classifier models in Gaia

### DISCUSSION THREAD

**Discussion by Galvo87** [MTG/essentia#1343] [Q&A]
Hi! Is there any updated resource about how to train your own SVM classifier models in Gaia, using Python bindings?
I found one but I suppose it's quite old and outdated.
Thanks

**Comment by palonso** [comment]
Hi @Galvo87,

We have not updated Gaia's python bindings nor their documentation recently since we are not maintaining the SVM models anymore.

What are the resources you found?  Do you have any specific problem?

**Comment by Galvo87** [comment]
Hi Pablo,

I found an old page, but I suppose it’s really outdated.
I am using Essentia with pleasure, but I would like to train my own
classifiers for tagging some unusual music tracks.
I don’t see much infos about how to do it, do you have any hint please?
Thank you,

Luca

On 29 June 2023 at 08:08:15, Pablo Alonso ***@***.***) wrote:

Hi @Galvo87 <https://github.com/Galvo87>,

We have not updated Gaia's python bindings nor their documentation recently
since we are not maintaining the SVM models anymore.

What are the resources you found? Do you have any specific problem?

—
Reply to this email directly, view it on GitHub
<https://github.com/MTG/essentia/discussions/1343#discussioncomment-6311651>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ADFWTKMYRQNXDZW4CULXRETXNULU7ANCNFSM6AAAAAAZNKVT7Q>
.
You are receiving this because you were mentioned.Message ID:
***@***.***>

**Comment by dbogdanov** [comment]
I guess you are referring to [this page](https://essentia.upf.edu/gaia_svm_models.html#training-your-own-svm-classifier-models-in-gaia). Let us know which parts are unclear.
